hs_core/models.py

```diff
@@ -604,7 +604,7 @@ def update(cls, element_id, **kwargs):
                             res_cr.order += 1
                             res_cr.save()
                     else:
-                        if res_cr.order > party.order:
+                        if res_cr.order > party.order and res_cr.order <= creator_order:
```

Comment:
`<=`  -> `<`?

Reply:
I'm 99% sure that we need the `<=` here because otherwise moving an author further down the list will cause a duplicate id.

Reply:
You are 100% right!

Reply:
thanks for checking!